region/client.go

```diff
@@ -359,7 +359,7 @@ func (c *client) readFully(buf []byte) error {
 func (c *client) sendHello(ctype ClientType) error {
 	connHeader := &pb.ConnectionHeader{
 		UserInfo: &pb.UserInformation{
-			EffectiveUser: proto.String("gopher"),
+			EffectiveUser: proto.String("root"),
```

Comment:
should we have an option to be able to specify this when we create a client instead?